thunder/__init__.py

```diff
@@ -575,15 +575,13 @@ def get_computation_and_inputs(*args, **kwargs):
                         arg_to_idx[a] = i
 
                     tensor_indices: int = []
-                    tensor_args_consumed_by_inplace_grouped_by_numel: dict[int, list[TensorProxy]] = defaultdict(list)
                     for bsym in filter(lambda b: b.sym.id == prims.PrimIDs.COPY_, computation_trc.bound_symbols):
                         t = bsym.flat_proxy_args[1]
                         index = arg_to_idx[t]
-                        numel = t.numel
-                        tensor_args_consumed_by_inplace_grouped_by_numel[numel].append(index)
                         tensor_indices.append(index)
-                    if len(tensor_args_consumed_by_inplace_grouped_by_numel) > 1:
-                        vanilla_tensor_args = set(tensor_indices)
+                    tmp_vanilla_tensor_args = set(tensor_indices)
```

Comment:
Comments please! This file is getting hard to read

<img src="https://media2.giphy.com/media/3TCNQbbz4tsP77wylp/giphy-downsized-medium.gif"/>

Reply:
oh sorry, this is just a smoke test. I should have added a do not review on the title first. :bow: